lib/srv/alpnproxy/conn_upgrade_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"crypto/tls"
 	"crypto/x509/pkix"
 	"errors"
+	apiclient "github.com/gravitational/teleport/api/client"
```

Comment:
nit: could you group imports.

Reply:
`make fix-imports` will fix this up and make the linter happy